Dockerfile

```
@@ -41,8 +42,9 @@ COPY --from=builder --chown=nextjs:nodejs /app/.next/standalone ./
 COPY --from=builder --chown=nextjs:nodejs /app/.next/static ./.next/static
 
 # Prisma CLI and schema — used by the Helm pre-upgrade migration Job
-COPY --from=deps --chown=nextjs:nodejs /app/node_modules/prisma ./node_modules/prisma
-COPY --from=deps --chown=nextjs:nodejs /app/node_modules/@prisma ./node_modules/@prisma
+# --follow-symlinks resolves pnpm virtual store symlinks so deps are real files in the image
+COPY --from=deps --follow-symlinks --chown=nextjs:nodejs /app/node_modules/prisma ./node_modules/prisma
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)